lib/cocoapods-rome/post_install.rb

```diff
@@ -183,6 +211,8 @@ def exclude_simulator_archs(installer)
   build_dir = sandbox_root.parent + 'build'
   destination = sandbox_root.parent + 'Rome'
 
+  nuke_frameworks(sandbox_root.parent, build_dir)
```

Comment:
I'd rename this to better represent what it's going to do:
```suggestion
  nuke_frameworks_if_needed(sandbox_root.parent, build_dir)
```